src/Magnum/DebugTools/ColorMap.cpp

```diff
@@ -373,11 +373,80 @@ constexpr UnsignedByte Viridis[][3] = {
     {246, 230,  31}, {248, 230,  33}, {250, 230,  34}, {253, 231,  36}
 };
 
+/* Taken from https://www.kennethmoreland.com/color-advice/ */
```

Comment:
Did you do this by hand? :) Like, even if you have a quick'n'dirty Python or whatever script, that'd be useful to preserve in case we'd like to add more from there, or the float versions and such.

In case you don't, there's no need to create it now, though.

Reply:
Sorry for the long delay. I used a python script, but its extremely hacky and just prints a pretty version of the csv file the website provides. Here it is in case you want to judge yourself : 

```

from numpy import genfromtxt
data = genfromtxt('cool_warm.csv', delimiter=',')
formatted = ''
data = data[1:,1:].astype(int)
formatted = ''
for i in range(256//4):
    line = ''
    for j in range(4):
        row = data[4*i+j]
        line += '{{{:>3}, {:>3}, {:>3}}}, '.format(row[0], row[1], row[2])
    formatted = formatted + line + '\n'

print(formatted)
```

Reply:
That's totally fine, actually ;) The main point of these is not to win code beauty competitions, but to *exist* so the next person doesn't have to write another one-off hacky script. Thank you.